src/components/Button/PlayButton.js

```diff
@@ -54,7 +53,7 @@ const PlayButton = ({ hover=false, type, expanded, className }) => {
 PlayButton.propTypes = {
   hover: PropTypes.bool,
   type: PropTypes.string,
-  expanded: PropTypes.oneOf([false, 'horizontal', 'vertical'])
+  expanded: PropTypes.oneOf(['horizontal', 'vertical'])
```

Comment:
Oh, I see - so it was supposed to be used here, I missed this. So it would be good to have it in it's own module. I'd just make a `src/components/utils/expansions.js` file that has `export default ['horizontal', 'vertical']` in it, then `CourseCard.js` and `PlayButton.js` can both `import expansions from '../../utils/expansions'` and use it.